utils_test.py

```diff
@@ -16,6 +16,7 @@
 
 # pylint: disable=relative-import
```

Comment:
Just a curiosity: 
1. why we can't make it oppia.utils?
2. Also is core.tests relative as well ?